news/2024/2024-01-28-vocaloid-mapping-contest-2.md

```diff
@@ -0,0 +1,175 @@
+| 2nd place | 2 months of osu!supporter, [*Elite Mapper* contest points](/wiki/Contests/Contest_points) |  |
+| 3rd place | 1 month of osu!supporter, [*Elite Mapper* contest points](/wiki/Contests/Contest_points) |  |
+
+\**Prize only for participants of the osu! game mode, due to its one-of-a-kind nature.*
+
+This contest is created by and for vocal synth fans to promote music, artists, and mappers. **All mappers who participate will receive constructive feedback from experienced mappers in the community.** That's arguably a more valuable prize in its own right!
+
+### Songs
+
+As proven by this contest, great things can happen when we promote and support the artists who make osu! great.
+
+Below we have selected 5 of iroha(sakaki)'s incredible works to you to pick from that were appropriate for contest mapping, and we're confident that something will inspire you!
+
+#### iroha(sasaki) - The Cuckoo Bird's Beautiful Youth (5:18 / 160 BPM)
+
+**Vocaloid Featured**: VY1 & Kagamine Rin
+
+**Lyricist**: π
+
+[Download the beatmap template here!](https://assets.ppy.sh/artists/399/Songs/iroha(sasaki)%20-%20The%20Cuckoo%20Bird_s%20Beautiful%20Youth.osz)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/399/Songs/iroha(sasaki)%20-%20The%20Cuckoo%20Bird_s%20Beautiful%20Youth.mp3" type="audio/mpeg">
+</audio>
+
+#### iroha(sasaki) - Shirley!! (4:59 / 165 BPM)
+
+**Vocaloid Featured**: Kagamine Rin
+
+**Lyricist**: π
+
+[Download the beatmap template here!](https://assets.ppy.sh/artists/399/Songs/iroha(sasaki)%20-%20Shirley!!.osz)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/399/Songs/iroha(sasaki)%20-%20Shirley!!.mp3" type="audio/mpeg">
+</audio>
+
+#### iroha(sasaki) - Devolution (4:26 / 190 BPM)
+
+**Vocaloid Featured**: Kagamine Rin
+
+**Lyricist**: kuma(alfred)
+
+[Download the beatmap template here!](https://assets.ppy.sh/artists/399/Songs/iroha(sasaki)%20-%20Devolution.osz)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/399/Songs/iroha(sasaki)%20-%20Devolution.mp3" type="audio/mpeg">
+</audio>
+
+#### iroha(sasaki) - 14 (Miku ver.) (4:55 / 128 BPM)
+
+**Vocaloid Featured**: Hatsune Miku
+
+**Lyricist**: π
+
+[Download the beatmap template here!](https://assets.ppy.sh/artists/399/Songs/iroha(sasaki)%20-%2014.osz)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/399/Songs/iroha(sasaki)%20-%2014.mp3" type="audio/mpeg">
+</audio>
+
+#### iroha(sasaki) - sigh (4:08 / 148 BPM)
+
+**Vocaloid Featured**: Kagamine Rin
+
+**Lyricist**: iroha(sasaki)
+
+[Download the beatmap template here!](https://assets.ppy.sh/artists/399/Songs/iroha(sasaki)%20-%20sigh.osz)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/399/Songs/iroha(sasaki)%20-%20sigh.mp3" type="audio/mpeg">
+</audio>
+
+### Rules
+
+- **Submissions must use one of the [selected songs](#songs).**
```

Comment:
mention if cut versions are allowed (so many people will ask with this song length)

Reply:
done